pkg/operator/ceph/csi/spec.go

```diff
@@ -129,6 +129,7 @@ var (
 	EnableRBD                 = false
 	EnableCephFS              = false
 	EnableNFS                 = false
+	EnableCSIOperator         = false
```

Comment:
All the callers will just access the new method `EnableCSIOperator()`, so this var should be private
```suggestion
	enableCSIOperator         = false
```